src/mob_prog.cpp

```diff
@@ -1023,14 +1023,14 @@ void mprog_translate(char ch, char *t, CHAR_DATA *mob, const CHAR_DATA *actor, c
 
     *t = '\0';
     switch (ch) {
-    case 'i': one_argument(mob->name, t); break;
+    case 'i': one_argument(mob->name.c_str(), t); break;
```

Comment:
haha, minimal work on mob_prog.cpp I see, fair doos, that is another rat's nest and I'm pretty skeptical about it right now.

Reply:
100% I suspect it's not long for this world so I didn't spend too much time making it better!

Reply:
Do I get a free spell or a customized item? :)